saleor/graphql/product/tests/test_product.py

```diff
@@ -950,6 +950,24 @@ def test_products_query_with_filter_stocks(
     assert {node["node"]["id"] for node in products_data} == product_ids
 
 
+def test_query_products_with_filter_ids(
+    api_client, product, query_products_with_filter
+):
+    # given
+    product_global_id = graphene.Node.to_global_id("Product", product.id)
+    variables = {"filter": {"ids": [product_global_id]}}
+
+    # when
+    response = api_client.post_graphql(query_products_with_filter, variables)
+
+    # then
+    content = get_graphql_content(response)
+    products_data = content["data"]["products"]["edges"]
+
+    assert len(products_data) == 1
+    assert products_data[0]["node"]["id"] == product_global_id
+
+
```

Comment:
We should have more products in this test to check if it really works. Right now we have only one product there so we don't know if it works correctly or not.

Reply:
Let's use `product_list` fixture and check if this filter returns 2 of 3 products.